guide/data-management-issues.rst

```diff
@@ -1,107 +1,127 @@
+    so is only recommended if you have a very large state vector that does not fit into memory on a single node.
 
-Filter run phases
------------------
 
-FO computation, prior and posterior
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    .. code-block:: text
 
-Run-time options include allocating spaces for two layouts and transposing
-between them, or running distributed in 'all copies' mode.
+        &state_vector_io_nml
+           buffer_state_io          = .true.,
+        /
 
 
-Assimilation
-~~~~~~~~~~~~
+Forward Operator (FO) computation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Distributed FO and QC observation ensembles
+The majority of calculation in an assimilation is done across the ensemble, for example,
+means, variances, increments, inflation, etc. and so the ideal data layout for assimilation is to have 
+all ensemble members for a given element of the state vector on the same MPI task. 
+However, :ref:`forward operator <FO>` calculations may need data from any part of the state
+vector, and so the ideal data layout for the forward operator is the entire state vector on 
+the same MPI task.
 
-Runtime option to either replicate the model state ensemble mean on each MPI task
-or run with that ensemble fully distributed.
+By default, DART runs in distributed mode, ``distributed_state = .true.`` (ideal for assimilation)
+where each MPI task has all the ensemble members for subset of the state vector.
+The forward operator calculation is vectorized across the whole ensemble, and DART takes 
+care of retrieving any state values needed for the forward operator calculation from other MPI
+tasks. 
 
+You may want to run in "transpose mode" where the first N MPI tasks perform the forward operator
+calculation, where N is the ensemble size. This is done by setting the namelist option
+``distributed_state = .false.`` in the filter_nml namelist. In general, this is not recommended
+as it requires more memory per task, has less vectorization, and does not scale beyond N tasks. 
+However, it may be useful
+for debugging.
 
-Ensemble memory usage and layout
---------------------------------
+Recommended setting for ``distributed_state``:
 
-Transposable
-~~~~~~~~~~~~
+.. code-block:: text
 
-Data is distributed over T MPI tasks but during the program execution
-the data is communicated between tasks to alternate between two different
-data layouts.
+    &filter_nml
+       distributed_state = .true.,
+    /
 
-Allocations are needed for two different 2D arrays: 
 
-* N ensemble members times (X items/T tasks)
-* X items times (N ensemble members/T tasks)
+Conversion of observation or state location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Distributed
-~~~~~~~~~~~
+The vertical location of an observation or state element may depend on the model state. For example, 
+an observation might be reported in pressure coordinates, requiring conversion based on the model state.
```

Comment:
"conversion to, for example, height" might help readers.

Reply:
fixed in ca2c67c43f85af9bc744c537aefbe3f65e786e0b